data/core/about.cfg

```diff
@@ -144,6 +144,10 @@
         comment = "coder, bug fixes"
     [/entry]
     [entry]
+        name = "Jonathan Combs (ScegfOd)"
```

Comment:
Quoting from https://wiki.wesnoth.org/PatchSubmissionGuidelines
"New contributors (i.e., those without commit access) should add themselves to the Miscellaneous Contributors section of the credits file (data/core/about.cfg), making sure to follow alphabetical order with respect to the first character that will be displayed."

Please move your entry to the appropriate section.